recipes/visan/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+export CMAKE_LIBRARY_PATH=$PREFIX/lib
+
+cmake $CMAKE_ARGS \
+  -DCMAKE_BUILD_TYPE="Release" \
+  -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+  -DCMAKE_PREFIX_PATH="$PREFIX" \
+  -DPython3_EXECUTABLE=$PREFIX/bin/python \
+  ..
+
+make -j$CPU_COUNT
+make install
```

Comment:
Is there a make check/test we can run here?

Reply:
No. This is not available. Visan is mostly a gui application. All the algorithmic elements are in the coda and harp packages that are already on conda-forge.

The tricky part is making sure everything links correctly properly (with the right versions of libraries, etc.), which is why the import tests in the meta file are the most important test for this package.